src/detection/Threshold.cc

```diff
@@ -115,7 +114,7 @@ double Threshold::getValue(ImageT const& image) const {
         math::Statistics stats = math::makeStatistics(image, math::STDEVCLIP);
         double const sd = stats.getValue(math::STDEVCLIP);
 
-        pexLogging::TTrace<3>("afw.detection", "St. Dev = %g", sd);
+        LOGLS_DEBUG("afw.detection.threshold", "St. Dev = " << sd);
```

Comment:
It looks like `LOGL_DEBUG("afw.detection.threshold", "St. Dev = %g", sd);` should also work, and that would be a smaller change from the original.